packages/codec/lib/special/decode/index.ts

```diff
@@ -203,3 +217,19 @@ function solidityVersionHasBaseFee(
       return true;
   }
 }
+
+function solidityVersionHasPrevrandao(
+  compiler: Compiler.CompilerVersion
+): boolean {
+  switch (Compiler.Utils.solidityFamily(compiler)) {
```

Comment:
do you think it may be useful to just add a comment here that Solidity versions above 0.8.18 have `block.prevrandao`?

Reply:
Hm, might be nice, yeah -- I could put it on all of these functions.  Not sure it's essential, might skip doing it unless I'm making some other change?